src/networking.c

```diff
@@ -1682,6 +1683,21 @@ void clientAcceptHandler(connection *conn) {
         }
     }
 
+    /* Auto-authenticate from cert_user field if set */
+    sds username = connGetPeerUsername(conn);
+    if (username != NULL) {
+        user *u = ACLGetUserByName(username, sdslen(username));
```

Comment:
We should probably check here if the user is enabled. We probably don't want to just authenticate straight into the user.